neural_compressor/config.py

```diff
@@ -399,9 +411,17 @@ def strategy(self):
     @strategy.setter
     def strategy(self, strategy):
         if check_value('strategy', strategy, str,
-            ['basic', 'mse', 'bayesian', 'random', 'exhaustive']):
+            ['basic', 'mse', 'bayesian', 'random', 'exhaustive', 'sigopt', 'tpe']):
```

Comment:
Please add `mse_v2` to the checklist.

Reply:
resolved:
66a238ba398dede70cb49e4a0d810d6939e54548